marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/deleteSavedQuery.sjs

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Pinging @ryanjdew on this - I am wondering what is needed for security here. Just occurred to me that we presumably at least need a check on the owner of the saved query being the same as the MarkLogic user that's connecting, right? Otherwise one user could delete anyone's saved query. 

@rahulvudutala Is there a role required for creating/deleting saved queries? If so, I believe we want an xdmp.securityAssert check here. I defer to @ryanjdew on whether we want to start requiring this now or wait a couple more stories, as @ryanjdew is getting a policy into place where we secure all of our DS endpoints with an xdmp.securityAssert check.

Reply:
@rjrudin 
1) I am checking in on the user here, 
let canDelete = cts.exists(cts.andQuery([cts.documentQuery(documentUri), cts.jsonPropertyValueQuery("owner", xdmp.getCurrentUser())]));
2) Yes. We have data-hub-saved-query-writer role with update permissions on the savedQuery document
Yes.

Reply:
Oops sorry, I missed that part of the query. 

@ryanjdew I defer to you if you want to get an xdmp.securityAssert in place here, or table it for now. We'll of course need to add those to a lot of existing endpoints at some point.